pkg/helm/kube.go

```diff
@@ -33,24 +36,34 @@ var kubeClient *kube
 var once sync.Once
 
 // GetConfigInstance returns a Kubernetes interface based on the current configuration
-func getConfigInstance() *kube {
+func getConfigInstance(kubeContext string) *kube {
 	once.Do(func() {
 		if kubeClient == nil {
 			kubeClient = &kube{
-				Client: getKubeClient(),
+				Client: getKubeClient(kubeContext),
 			}
 		}
 	})
 	return kubeClient
 }
 
-func getKubeClient() kubernetes.Interface {
-	kubeConf, err := config.GetConfig()
-	if err != nil {
-		fmt.Println("Error getting kubeconfig:", err)
+func getKubeClient(kubeContext string) kubernetes.Interface {
+
+	var kubeConfig *rest.Config
+	var kubeConfError error
+
+	if kubeContext == "" {
+		kubeConfig, kubeConfError = config.GetConfig()
+	} else {
+		klog.V(3).Infof("Using kube context: %s", kubeContext)
+		kubeConfig, kubeConfError = config.GetConfigWithContext(kubeContext)
```

Comment:
I'm pretty sure you can do this without needing to check for an empty string above. If the `kubeContext` var is an empty string it will just return the current context of the kubeconfig.

I used similar code here: https://github.com/FairwindsOps/reckoner/blob/master/pkg/reckoner/client.go#L111

Reply:
Cool.